### PR TITLE
Made few changes TC0008, now it is up and running.

### PR DESCRIPTION
**Pull request by poonam-94**
Poonam-made changes in TC0008, now it is up and running

### CODE REVIEW

File: Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java [hunk 1]
```diff
@@ -1,29 +1,29 @@
-package pageobjects;
-
-import java.util.List;
```
Comment (akashdktyagi) [inline Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java:3]: There is some mistake in the way you guys are making commiting. pull request should only display the difference of lines, not the entire file deleted and added.

File: Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-
-import org.junit.Assert;
-import org.openqa.selenium.By;
```
Comment (akashdktyagi) [inline Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java:6]: Test Comment.

File: Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-
-import cucumber.api.Scenario;
```
Comment (akashdktyagi) [inline Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java:10]: asdsfdd

File: Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java (hunk 1 above)
Comment (poonam-94) [reply Batch8CucumberSeleniumAPI/src/test/java/pageobjects/ProductDescriptionObjects.java:3]: Ok sir,  i will resolve it.

**Review by akashdktyagi — COMMENTED** [review]
na

**Review by akashdktyagi — CHANGES_REQUESTED** [review]
na

**Pull request closed without merging**